src/kernels/linux-hardened.sh

```diff
@@ -64,9 +64,9 @@ update_linux_hardened_pkgbuilds() {
 update_linux_hardened_git_pkgbuilds() {
     pkg_list=("spl-linux-hardened-git" "zfs-linux-hardened-git")
     kernel_version=${kernel_version_git}
-    kernel_version_full=$(kernel_version_full ${kernel_version_git})
-    kernel_version_full_pkgver=$(kernel_version_full_no_hyphen ${kernel_version_git})
-    kernel_version_major=${kernel_version_git%-*}
+    kernel_version_full=$(kernel_version_full ${kernel_version})
+    kernel_version_full_pkgver=$(kernel_version_full_no_hyphen ${kernel_version})
+    kernel_version_major=${kernel_version%-*}
```

Comment:
This does not change the pkgver,  `kernel_version` is set to `kernel_version_git` on top:
`kernel_version=${kernel_version_git}`
This way you can copy any instructions made in the none git functions, and don't have to change every occurrence of `kernel_version`. It's only a minor change